### ISSUE TITLE
Include geoparquet download links as assets in collection metadata

### ISSUE THREAD

**Issue by VincentVerelst** [WorldCereal/ewoc_rdm_api#15]
Currently for a certain collection_id the download links of the raw and sampled geoparquet files are found at `https://ewoc-rdm-api.iiasa.ac.at/collections/<collection_id>/download` and `https://ewoc-rdm-api.iiasa.ac.at/collections/<collection_id>/sample/download` respectively. 

Is it possible to include both datasets as assets in the collection metadata? This way both dataset download links can be found immediately at `https://ewoc-rdm-api.iiasa.ac.at/collections/<collection_id>`

Here is an example of collection metadata with assets: https://github.com/fiboa/fiboa.github.io/blob/main/stac/datasets/ukfields.json

**Comment by santoshkaranam** [comment]
This API already exists.  Example https://ewoc-rdm-api.iiasa.ac.at/collections/2017afoneacrefundmelpoint110/metadata/items

![image](https://github.com/user-attachments/assets/e9a52dcf-7618-4c20-ac53-9978a324787a)

use name to locate download urls in the list - GeoParquetDownloadUrl, SampleDownloadUrl
database Ids may change, so only use "name"

**Comment by kvantricht** [comment]
To be checked with @jdries if this is still needed.

**Comment by jdries** [comment]
@santoshkaranam If I'm correct, this is a custom API right?
The idea in the original request was to follow the STAC standard. By doing that, we could more easily work with existing tools, and integrate in openEO in a standardized manner.
Of course, WorldCereal toolbox is able to write custom code that handles this API as well if needed.

**Comment by santoshkaranam** [comment]
Yes, We have an issue for this tracked. STAC will be mainly to expose datasets. For uploading and for signed URLs for private dataset, we will still use this API

**Comment by jdegerickx** [comment]
#16
